### PR TITLE
Implement the linter strategy starting by LSP

### PR DESCRIPTION
**Pull request by Nexucis**
lsp-linter

/cc @slrtbtfs 

Offline linter will come later

### CODE REVIEW

**Comment by Nexucis** [comment]
Related to #5

File: src/app/app.ts [hunk 1]
```diff
@@ -66,46 +66,74 @@ const basicSetup: Extension = [
...
-  activateLSPAutocompletion();
-});
+function applyConfiguration(): void {
+  setCompletion();
+  setLinter();
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+  // @ts-ignore
+  // When the linter is changed, it required to reload completely the editor.
+  // So the first thing to do, is to completely delete the previous editor and to recreate it from scratch
+  document.getElementById('editor').innerHTML = '';
```
Comment (juliusv) [inline src/app/app.ts:129]: Might be nicer to save the `EditorView` somewhere and call `view.destroy()` on it here vs. just setting the `innerHTML` to empty?

**Comment by juliusv** [comment]
:+1: besides comment

File: src/app/app.ts (hunk 1 above)
Comment (Nexucis) [reply src/app/app.ts]: yup definitely better. Didn't know it exists ( and also I didn't search it my bad )

Thanks for the info :)

**Comment by Nexucis** [comment]
> It might be worth investigating whether it's possible to additionally show those error messages below the editor, without having to hover over the red lines.

No idea if it's possible, but otherwise it's how the IDE are working in generally, and I didn't think it was so bad. But maybe it's just me hahaha

**Pull request merged**